cloud/datasync/test/SyncServerTest.ts

```diff
@@ -74,3 +76,42 @@ describe('FeedHenry Sync Tests', function() {
     });
   });
 });
+
+@suite
```

Comment:
These annotations are actually mocha-typescript stuff. -1 from me from the other comment. It won't allow us to have a good support for our test suites to be used by non-TS code.

In addition it suggests a different naming that what we usually go for when using mocha's default BDD interface.
i.e. instead of `SyncServerTest#testConnect()` we would usually have `'SyncServer' ... 'should be able to connect'